functions/api_status/main.py

```
@@ -0,0 +1,31 @@
+import json
+import os
+from datetime import datetime
+
+from google.cloud import firestore
+
+
+def api_status(request):
+    """Cloud Functions用 APIステータスエンドポイント"""
+    # Firestore接続確認
+    try:
+        db = firestore.Client()
```

Ruff (0.12.2)

Local variable `db` is assigned to but never used

Remove assignment to unused variable `db`

(F841)